src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -567,7 +567,7 @@ public static LbfgsLogisticRegressionBinaryTrainer LbfgsLogisticRegression(this
         /// <example>
         /// <format type="text/markdown">
         /// <![CDATA[
-        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/PoissonRegression.cs)]
+        ///  [!code-csharp[PoissonRegression](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LbfgsPoissonRegression.cs)]
```

Comment:
>PoissonRegression [](start = 27, length = 17)

?

Reply:
I don't think we need to prefix this with Lbfgs.

---
In reply to: [270581846](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270581846) [](ancestors = 270581846)

Reply:
This is the alt-text for the link, so it should be the name of the API.

---
In reply to: [270973957](https://github.com/dotnet/machinelearning/pull/3140#discussion_r270973957) [](ancestors = 270973957,270581846)